docs/DEVELOPER_GUIDE.md

```diff
@@ -191,3 +191,30 @@ controller is simply:
 ```shell
 make deploy-dev
 ```
+
+## Troubleshooting
+
+1. When you run make deploy, you may encounter an error like this:
```

Comment:
Can we use a ### for this? It will help our troubleshooting doc scale.

Perhaps


### Make Deploy, `error: error validating "STDIN": error validating data`
```shell
...
```

Reply:
It will add it in the top generated TOC, which will look ugly...